styles.css

```
@@ -0,0 +1,235 @@
+
+  .header-photo {
+    width: 6rem;
+    height: 6rem;
+  }
+
+  h2,
+  h3,
+  h4,
+  .job-dates {
+    break-after: avoid-page;
+    page-break-after: avoid;
```

Stylelint (17.14.0)

[error] Expected "page-break-after" to be "break-after" (property-no-deprecated)

(property-no-deprecated)